src/test/java/uk/gov/hmcts/reform/ccd/util/log/CaseDataViewBuilderTest.java

```diff
@@ -1,76 +1,32 @@
 package uk.gov.hmcts.reform.ccd.util.log;
 
 import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.ccd.data.model.CaseData;
 import uk.gov.hmcts.reform.ccd.data.model.CaseDataView;
-import uk.gov.hmcts.reform.ccd.data.model.CaseFamily;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static uk.gov.hmcts.reform.ccd.fixture.TestData.DELETABLE_CASE_FAMILY;
-import static uk.gov.hmcts.reform.ccd.fixture.TestData.DELETABLE_CASE_FAMILY_SIMULATION;
+import static uk.gov.hmcts.reform.ccd.fixture.TestData.DELETABLE_CASE_TYPE;
 import static uk.gov.hmcts.reform.ccd.util.LogConstants.DELETED_STATE;
-import static uk.gov.hmcts.reform.ccd.util.LogConstants.FAILED_STATE;
-import static uk.gov.hmcts.reform.ccd.util.LogConstants.SIMULATED_STATE;
 
 class CaseDataViewBuilderTest {
     @Test
     void shouldCreateCaseDataViewForDeletableCases() {
```

Comment:
Please either extend this test or create a new ones to cover simulated and failed states